subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -46,10 +48,18 @@ LOG_MODULE_REGISTER(bt_ctlr_isoal, LOG_LEVEL_INF);
 
 #include "hal/debug.h"
 
+#define TRUE         1U
+#define FALSE        0U
```

Comment:
I am defining these because there are bit fields used as boolean packed into the unused bits of a uint64 that only uses 39 bits. The code is more readable if the assignment is either true or false. As I understand it using the bool type definitions with the bit fields makes our compiler unhappy.

I didn't see common definitions that I could reuse but if there is a standard approach that I missed, please let me know.

Reply:
That's also what I thought. I do, however, think that this should be a generic approach for the controller, and I would suggest to (if you really want to add this) move this to a new PR that defines these a common place in the controller (if we don't already have anything like this), and apply it everywhere in the controller where we use a bit in a bitfield as a boolean value. 

I do not think this should be defined in a single .c file and only applied there

Reply:
Created a new PR https://github.com/zephyrproject-rtos/zephyr/pull/61591 for the definitions.

Reply:
The conclusion reached in the controller meeting today was that definition of `TRUE` and `FALSE` was unnecessary. So as agreed I have reverted to use 1U and 0U instead.